assets/css/style.css

```diff
@@ -137,7 +275,7 @@ html[dir='rtl'] .clearButton {
     top: .7em;
 }
 
-.apertiumSubLogo + .apertiumLogo {
+.apertiumSubLogo+.apertiumLogo {
```

Comment:
Why did you remove this standard binary spacing?

Reply:
We should really create a .editorconfig file. I just used VSCode's beautify plugin to do that.

Reply:
Feel free to do it and send a PR.